packages/css/src/components/screen/screen.scss

```diff
@@ -3,7 +3,7 @@
  * Copyright Gemeente Amsterdam
  */
 
-@mixin reset {
+@mixin reset-div {
```

Comment:
Hmm… what to do with this one. You could argue this is not even a reset. We’re not overriding some opiniated design of the browser, we’re just setting a different value for this keyword. Just include it in the main selector?